protos/feast/core/Store.proto

```diff
@@ -110,6 +110,8 @@ message Store {
   message RedisConfig {
     string host = 1;
     int32 port = 2;
+    int32 backoff_ms = 3;
```

Comment:
Suggest some comments to the new proto fields.
```suggestion
    // Optional. The number of milliseconds to wait before retrying failed Redis connection. 
    // By default, Feast uses exponential backoff policy and "backoff_ms" sets the initial wait duration.
    int32 backoff_ms = 3;
    
    // Optional. Maximum total number of retries for connecting to Redis. Default to zero retries.
    int32 max_retries = 4;
```

Reply:
Just nitpick, the initial backoff duration could be renamed as follow
```
int32 initial_backoff_ms = 3;
```
So that the generated code will be clearer

Reply:
@davidheryanto @pradithya Thanks for the suggestion, i have implemented the change as suggested.